### ISSUE TITLE
Freeze when interrupting a job

### ISSUE THREAD

**Issue by Cheaterman** [scopatz/xonsh#1021]
Hello,

xonsh seems to consistently freeze the second time I'm interrupting a job, as the following screenshot demonstrates:

http://i.imgur.com/KHzYrG0.png

I'm currently running the latest 0.3.0 and am still affected. My term is rxvt-unicode if that matters.

Thanks in advance! Xonsh is awesome :-)

**Comment by adqm** [comment]
Thanks for reporting, @Cheaterman!  I can't reproduce this personally.  Can you run the following command and paste the input here?

```
$ xonfig
```

**Comment by Siecje** [comment]
I just installed 0.3.0 in a new ```virtualenv``` and I can reproduce.

**Comment by scopatz** [comment]
I find it curious that yours says 'stopped' after the first `^Z`.  Mine works fine and does not say this:

```console
scopatz@artemis ~ $ sleep 10
^Z
scopatz@artemis ~ $ fg
[1] running: sleep 10 (3688)
^Z
scopatz@artemis ~ $ 
```

This makes me think that the process is getting sent something other than SIGTSTP

**Comment by Cheaterman** [comment]
Thanks @scopatz for your answer! xonsh development is going too fast for me hehehe. Apparently, this issue is fixed in 0.3.2 - indeed just like you I won't get "stopped" after the ^Z, but I'll still get "running" on fg like you do. I'm closing this, thanks a lot to all of xonsh dev team!
